src/utils/axios_interceptor.tsx

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: '',
+  timeout: 1000,
+});
+
+axios.interceptors.request.use(
```

Comment:
```suggestion
api.interceptors.request.use(
```

axios.interceptors가 아닌 위에 정의한 api를 가져와야 할 것 같아요..!

Reply:
엇! 변경했습니다.... ㅎㅎ

하다가 왜 `axios`로 작성했는데도 에러가 나지 않았을까? 를 생각하고 공부한 결과입니다.

- `api` 자체가 `axios.create`로 생성된 axios 인스턴스 때문
- `api.interceptors`로 작성시
  - 해당 인스턴스에서 발생하는 모든 요청에 인터셉터가 적용됨
- `axios.interceptors`로 작성시
  - axios를 사용하는 모든 요청에 인터셉터가 적용됨 
  (`axios.interceptors.request.use` 는 **글로벌** 인터셉터로 설정되기 때문)